main.js

```diff
@@ -1 +1,20 @@
-console.log('Hello World');
\ No newline at end of file
+const level = require('level');
+const db = connectToDatabase('./studentdb');
+
+function connectToDatabase(db){
+    return level(db, { valueEncoding: 'json' });
+}
+
+var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
+
+acceptStudent('201812485', 'Abdul Moiz Solaiman', 22, 'Marawi City');
+
+function acceptStudent(id, fullName, age, address){
+    let attributes = { ID:id, Name: fullName, Age: age, Address: address };
+    db.put(id, attributes, function(err){
```

Comment:
you don't need to use the promise base approach for the functions minimizing complexity...

Reply:
it has been updated sir.